src/main/bin/ccm-fetch

```diff
@@ -245,9 +249,16 @@ $fetch->setPreprocessor($this_app->option("preprocessor"))
 $fetch->setCacheRoot($this_app->option("cache_root"))
 		if defined $this_app->option("cache_root");
 
-# set world readable flag
-$fetch->setWorldReadable($this_app->option("world_readable"))
-		if defined $this_app->option("world_readable");
+# set group/world readable flag
+if (defined $this_app->option("group_readable")) {
+    $fetch->setWorldReadable($this_app->option("group_readable"));
+    if ($this_app->option("world_readable")) {
+        $this_app->info("group_readable set, not setting world_readable");
```

Comment:
Ignoring an option because something else is set seems wrong. If world_readable is set then that should take precedence over group_readable?